app/presenters/result_set_presenter.rb

```diff
@@ -49,10 +50,27 @@ def selected_filter_descriptions
     }.reject(&:empty?)
   end
 
+  def is_best_bet?(score_a, score_b)
```

Comment:
This method can also access `results` so we don't need to pass anything to it.

The checks for whether  an `es_score` exists could be moved here as well. Something like:

```
score_a = results[0].es_score
score_b = results[1].es_score

# We found the average score on the top 500 searches and found that 7 was the most suitable number
if score_a.present? && score_b.present?
  (score_a / score_b) > 7
end
```

In Ruby convention you don't need the `is_` in the method name as that's covered by the `?`.
Also, this method should be private.